pgjdbc/src/main/java/org/postgresql/util/OSUtil.java

```diff
@@ -26,7 +26,7 @@ public static boolean isWindows() {
    */
   public static String getUserConfigRootDirectory() {
     if (isWindows()) {
-      return System.getenv("APPDATA") + File.separator;
+      return System.getenv("APPDATA") + File.separator + "postgresql" + File.separator;
```

Comment:
Just wondering: is there a reason for the directory to include the trailing `File.separator`?

Reply:
the next thing we do is add a file to the end of it, so we might as well add the separator here.

Reply:
There's an abstraction leak: directory name does not include the separator (the slash is not a part of the name).

Reply:
we could change the name of the function or remove the separator. This is really only used in 2 places (at the moment) so I'm ambivalent as to which we do